### PR TITLE
Remove obsolete files related to rdkit-js + write a rescoped README for MinimalLib

### PR DESCRIPTION
**Pull request by MichelML**
#### Reference Issue
https://github.com/rdkit/rdkit-js/issues/130

#### What does this implement/fix? Explain your changes.
Wrote a rescoped readme in the minimallib source code post release of the official rdkit-js repository

#### Any other comments?
cc @ptosco @greglandrum, let me know if you think this makes sense :+1:

### CODE REVIEW

**Review by greglandrum — APPROVED** [review]
The changes look ok to me.

At some point we should update the MinimalLib docs to make them a bit less JS-specific. The primary idea of the library is to make it easy to use the RDKit from any language where we don't want to do explicit wrappers, not just to provide JS support (even though JS will likely be the most frequently used).

**Comment by greglandrum** [comment]
Will wait for @ptosco's comments before merging.

**Comment by MichelML** [comment]
while @ptosco finds the time to check this, I made further replacement of rdkit-js links in the rdkit docs [here](https://github.com/rdkit/rdkit/pull/5432/commits/fade6cbb74dabe67167f504f46bb13029ad3f69c)

a bit out of scope, but I added the UGM 2021 link in the main README [here](https://github.com/rdkit/rdkit/pull/5432/commits/a34a2faf919a6b1075d74f8bc03b2f44fcc018fa)

File: Code/MinimalLib/README.md [hunk 1]
```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
...
 
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/demo.html
+If you are using the MinimalLib for the first time, see the getting started examples at https://www.rdkitjs.com/ .
 
-## Contributing
+### All live demos
 
-### Preparing a new release of the package
+- RDKit.js website: https://www.rdkitjs.com/
```
Comment (ptosco) [inline Code/MinimalLib/README.md:101]: This is not live yet, is it?

File: Code/MinimalLib/README.md [hunk 2]
```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
...
+- RDKit.js usage with React.js: https://react.rdkitjs.com/
```
Comment (ptosco) [inline Code/MinimalLib/README.md:102]: Ditto

File: Code/MinimalLib/README.md [hunk 3]
```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
...
-Make sure you are at the root of the [RDKit](https://github.com/rdkit/rdkit) GitHub project, and on the branch and version of the project you want to release. **Note that no commits should occur during the release process.**
+## Building the MinimalLib
 
-#### Step 1: Set the release version in package.json
+Make sure you are at the root of the [MinimalLib](https://github.com/rdkit/rdkit/tree/master/Code/MinimalLib), and run the following script.
 
 ```bash
-npm --no-git-tag-version version <semver version matching an RDKit release>
-# Example npm --no-git-tag-version version 2021.3.1
+bash scripts/build_rdkitjs.sh <RDKit git release tag name>
```
Comment (ptosco) [inline Code/MinimalLib/README.md:111]: Does this script exist?

**Review by ptosco — CHANGES_REQUESTED** [review]

File: Code/MinimalLib/README.md (hunk 1 above)
Comment (MichelML) [reply Code/MinimalLib/README.md:101]: it is!

This is a combination of @greglandrum initial tutorials + the react examples I implemented + css styling + cross reference to related websites (rdkit, npm, rdkit-js source code)

I currently own the domain on Vercel but happy to transfer it to anyone

File: Code/MinimalLib/README.md (hunk 3 above)
Comment (MichelML) [reply Code/MinimalLib/README.md:111]: yep, see here https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/scripts/build_rdkitjs.sh

File: Code/MinimalLib/README.md (hunk 2 above)
Comment (MichelML) [reply Code/MinimalLib/README.md:102]: there are two different links here, one for vanilla JS examples https://www.rdkitjs.com/ , and one for React examples https://react.rdkitjs.com/

File: Code/MinimalLib/README.md (hunk 1 above)
Comment (ptosco) [reply Code/MinimalLib/README.md:101]: @MichelML Ah, right - I can access it from my personal phone but not from the corporate network. I'll submit a request to whitelist it. Sorry for the false alarm!

**Comment by MichelML** [comment]
that's my dream @ptosco , see a related comment in your repository here https://github.com/rdkit/rdkit-structure-renderer/pull/2#issuecomment-1179290487

maybe we should open an official discussion topic to establish a plan on how to centralize all rdkit-js efforts?

File: Code/MinimalLib/README.md (hunk 2 above)
Comment (MichelML) [reply Code/MinimalLib/README.md:102]: that's my dream @ptosco :smile:  , see a related comment in your repository here https://github.com/rdkit/rdkit-structure-renderer/pull/2#issuecomment-1179290487

maybe we should open an official discussion topic to establish a plan on how to centralize all rdkit-js efforts?

**Comment by MichelML** [comment]
> https://github.com/rdkit/rdkit/pull/5432#issuecomment-1190397594

No rush @ptosco but yes we should continue the conversation whenever we find the time

**Pull request merged**